### ISSUE TITLE
Sniping doesn't work outside of touch range

### ISSUE THREAD

**Issue by booman246** [TVPT/VoxelSniper#333]
Expected: 
Using the arrow, right clicking should apply a brush up to the range set by /vs range <range>

Issue:
Brushes are only applied when inside of touch range (ie. the bounding box appears around a block). Brushes are not applied at longer distances even if /vs range is expanded to a much larger number than the current distance (ie 100)

Additional Information:
Sponge: spongevanilla-1.12.2-7.1.4
Minecraft: 1.12 Vanilla
VoxelSniper: 8.5.0 SNAPSHOT build 478 on Jenkins http://ci.voxelmodpack.com/job/VoxelSniper/478/

**Comment by booman246** [comment]
Have a little bit of further information here. I'm now led to believe that this is a Sponge issue or change to the API as it also affects Worldedit brushes and farwand.

I did a bit of legwork to figure out where the change occurs. The last release that works without this issue is SpongeVanilla 1.12.2-7.1.0-BETA-113. The issue first appears in 1.12.2-7.1.0-BETA-114.

I'm not sure if this is an issue with Sponge, or an API change that VS hasn't been updated for. If it's a Sponge issue, I can file over on the Sponge github instead.

**Comment by booman246** [comment]
The following commit is where the bump from BETA-113 to BETA-114 occurred. It looks to me like removing the right click handler in the MixinPlayerInteraction is what has caused this change to occur.

I'm not sure if this is a deprecated function and VS needs to be updated, or if removing the function was an error.

https://github.com/SpongePowered/SpongeVanilla/commit/3ead93f760bdcfe51771593870964862d92ba0f3#diff-0fd449e25c319b13b3cef0db05c6af9cL24

**Comment by Deamon5550** [comment]
We're correctly using the `InteractItemEvent` which was created to cover this use case. I would consider this a sponge bug not throwing this event in all cases (this isn't the first time this has happened). Please open an issue on SpongeCommon referencing this.

**Comment by ImMorpheus** [comment]
> I would consider this a sponge bug not throwing this event in all cases (this isn't the first time this has happened).

Correct.
Right-click on AIR with an itemstack is not firing an event on both SF and SV.
However Forge is firing its own PlayerInteractEvent which is converted to InteractItemEvent, so this shouldn't be an issue on SpongeForge.

**Comment by gabizou** [comment]
Fixed in latest versions of SpongeForge and SpongeVanilla.

**Comment by StreakDragon** [comment]
Can anyone help me, I've been trying to get VoxelSniper on my Sponge Server for a while now, everywhere I go I keep getting bad gateways and with the Jenkins link I signed up and it tells me I don't have overall read permissions...every turn I take I get thwarted and can not get 8.5.0 (build 478?)  I have sponge version 7.1.6 RC3616.  Can SOMEONE help me here.  

Thank you in advance for anyone that can either provide me the file or point me to a working site.  I also tried the VoxelSniper IRC Channel (why aren't they using Discord???) and literally no one has answered, spoke or anything else for over 3 days so I'm absolutely stuck.
